go-controller/pkg/ovn/external_ids_syncer/acl/acl_sync_test.go

```diff
@@ -73,7 +73,25 @@ func joinACLName(substrings ...string) string {
 
 func buildPortGroup(hashName, name string, ports []*nbdb.LogicalSwitchPort, acls []*nbdb.ACL) *nbdb.PortGroup {
 	externalIds := map[string]string{"name": name}
-	return libovsdbops.BuildPortGroup(hashName, ports, acls, externalIds)
+	pg := nbdb.PortGroup{
```

Comment:
self-note: ah we need to keep a copy of the `BuildPortGroup` still for older updates...